app/frame/directives/mode-bar.js

```diff
@@ -20,9 +22,20 @@ function (
                 $scope.activeView = view_name;
             };
 
+            $scope.loggedin = $rootScope.loggedin;
```

Comment:
I believe you'll find this doesn't actually work in some cases since it won't update when state changes. Can you use the previous bindings instead? ie. `ng-if="currentUser"`